report-ui/style.css

```
@@ -738,46 +793,179 @@ select:focus {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  max-width: calc(100% - 160px);
+}
+
 .graph-back-btn {
+  gap: 8px;
+  padding: 8px;
+  background: rgba(15, 23, 42, 0.7);
+  border: 1px solid rgba(100, 116, 139, 0.25);
+  border-radius: 12px;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "border-radius" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)